devise.rb

```diff
@@ -165,28 +162,31 @@ def home
   # Webpacker / Yarn
   ########################################
   run 'yarn add bootstrap @popperjs/core'
-  run "rails webpacker:install:stimulus"
-  append_file 'app/javascript/packs/application.js', <<~JS
+  append_file 'app/javascript/application.js', <<~JS
     import "bootstrap"
   JS
 
-  inject_into_file 'config/webpack/environment.js', before: 'module.exports' do
-    <<~JS
-      // Preventing Babel from transpiling NodeModules packages
-      environment.loaders.delete('nodeModules');
-    JS
-  end
+  gsub_file 'package.json', /(\s}){2}/, '},
+"scripts": { "build": "webpack --config webpack.config.js" }
+}'
+
```

Comment:
Not sure we need this, I got this line added twice with both templates: 

![image](https://user-images.githubusercontent.com/57643651/155580782-d1e8370e-4848-4748-b2ab-0a94784cc60f.png)

Reply:
As we saw together this is fixed when using node >16 like we do in the setup 👏